scripts/gen_apitools_client.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+pip install --upgrade google-apitools[cli]
+
+TMPDIR="$(mktemp -d)/storage"
+gen_client --discovery_url=storage.v1 --outdir=${TMPDIR} \
+--user_agent= --root_package=gcloud.storage \
+--client_id= --client_secret= --nogenerate_cli client
+mv ${TMPDIR}/storage_v1_* gcloud/storage/
+
+PREV_IMPORT="import storage_v1_messages as messages"
```

Comment:
actually, you should be able to just pass the value you want here for `--root_package` and not have to `sed`.